bundler/lib/bundler/cli/info.rb

```diff
@@ -14,6 +14,20 @@ def run
         Bundler.load.lock
       end
 
+      if gem_name
+        if gem_name == "bundler"
+          path = File.expand_path("../../../..", __FILE__)
+        else
+          spec = Bundler::CLI::Common.select_spec(gem_name, :regex_match)
+          return unless spec
+          path = spec.full_gem_path
+          unless File.directory?(path)
+            return Bundler.ui.warn "The gem #{gem_name} has been deleted. It was installed at: #{path}"
+          end
+        end
+        return Bundler.ui.info(path)
```

Comment:
Hi @devraj-kumar! Thanks for keeping on helping us! :heart:

I see that the CI is complaining because our specs expect a bunch of output to be printed by `bundle info`, but now only the path is being printed. That suggests that you at least need to remove this early `return`, and simply print the information and go on. Something like:

```suggestion
        Bundler.ui.info(path)
```

However, though, it seems that that "path information" is being printed at the `print_gem_path` method further down. So I believe the check about directory existence should be moved to there.

Let me know if this information allows you to move on or if you need further help.